src/GraphicsLayer.ts

```diff
@@ -22,7 +23,7 @@ module PIXI.tilemap {
             renderer.context.globalAlpha = 1.0;
         }
 
-        renderWebGL(renderer: WebGLRenderer) {
```

Comment:
@ivanpopelyshev  , when use module PIXI.tilemap , the `PIXI.` is optional.
but if use `namespace pixi_tilemap` ,  the `PIXI.`  is required.